File: src/main/java/net/noncore/fdx/views/MainView.java
package net.noncore.fdx.views;

import javafx.fxml.FXML;
import javafx.scene.layout.AnchorPane;
import net.noncore.fdx.views.components.FilesBoard;

public class MainView extends AnchorPane implements View {
    @FXML
    private FilesBoard filesBoard;

    public MainView() {
        load("MainView.fxml");
    }

}

File: src/main/kotlin/net/noncore/fdx/views/MainView.kt
package net.noncore.fdx.views

import javafx.scene.layout.AnchorPane
import tornadofx.*

class MainView : View("Main View") {
    override val root: AnchorPane by fxml("/MainView.fxml")

    override fun onCreate() {
        super.onCreate()
    }

    override fun onBeforeShow() {
        super.onBeforeShow()
        root.children.add(FilesBoard().root)
    }
}
